brightray/browser/url_request_context_getter.cc

```diff
@@ -138,6 +139,7 @@ URLRequestContextGetter::URLRequestContextGetter(
       in_memory_(in_memory),
       io_task_runner_(io_task_runner),
       file_task_runner_(file_task_runner),
+      ct_delegate_(new RequireCTDelegate),
```

Comment:
This creates the delegate on the UI thread, we should move the initialization into `GetURLRequestContext` method below.

Reply:
:+1: